inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/thirdparty/sort/util/SinkInfoUtils.java

```diff
@@ -153,59 +141,29 @@ private static HiveSinkInfo createHiveSinkInfo(HiveSinkResponse hiveInfo, List<F
         } else {
             fileFormat = new HiveSinkInfo.TextFileFormat(separator);
         }
-        // The primary partition field, in Sink must be HiveTimePartitionInfo
-//        List<HivePartitionInfo> partitionList = new ArrayList<>();
-//        String primary = hiveInfo.getPrimaryPartition();
-//        if (StringUtils.isNotEmpty(primary)) {
-//            // Hive partitions are by day, hour, and minute
-//            String unit = hiveInfo.getPartitionUnit();
-//        HiveTimePartitionInfo timePartitionInfo = new HiveTimePartitionInfo(
-//                primary, PARTITION_TIME_FORMAT_MAP.get(unit));
-//            partitionList.add(timePartitionInfo);
-//        }
-        // For the secondary partition field, the sink is temporarily encapsulated as HiveFieldPartitionInfo,
-        // TODO the type be set according to the type of the field itself.
-//        if (StringUtils.isNotEmpty(hiveInfo.getSecondaryPartition())) {
-//            partitionList.add(new HiveSinkInfo.HiveFieldPartitionInfo(hiveInfo.getSecondaryPartition()));
-//        }
 
         // Handle hive partition list
-        List<HivePartitionInfo> partitionList;
-        if (CollectionUtils.isNotEmpty(hiveInfo.getPartitionFieldList())) {
-            checkPartitionField(hiveInfo);
-            hiveInfo.getPartitionFieldList().sort(Comparator.comparing(HivePartitionField::getRankNum));
```

Comment:
Whether the sorting by removing the rank order is as expected?

Reply:
Currently, we do not use the rankNum to order the fields.